test/ssl_certs/localhost.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Added a test for verify_ssl - for which I needed a cert with a valid common name.